src/qappfw.js

```diff
@@ -305,22 +308,35 @@ class QRadar {
      * @throws Error if aql is not supplied or if the search results could not be displayed.
      *
      */
+    
     static openEventSearch(aql, openWindow)
     {
         if (aql == null)
         {
             throw new Error("You must supply an AQL string");
         }
-
-        return QRadar.windowOrTab(
-            "do/ariel/arielSearch?appName=EventViewer&pageId=EventList&dispatch=performSearch" +
-            "&value(timeRangeType)=aqlTime&value(searchMode)=AQL" +
-            "&value(aql)=" + encodeURIComponent(aql), openWindow === false ? "EVENTVIEWER" : null);
+        // Creates variables for fetching qradar version, removing the dots and taking the first 6 characters from it, i.e. 202164
+        var get_qradar_version = top.QRADAR_VERSION.replaceAll(".","");
```

Comment:
For JavaScript (and this project) we normally use camelCase instead of snake_case, so just for code style could you update any variables that are `snake_case` to be `camelCase`.